src/app/components/engangsstonad/steps/EngangsstonadStep3.js

```diff
@@ -42,13 +43,13 @@ export class Step4 extends Component {
 
 		this.props.postEngangsstonadToApi({
```

Comment:
Litt rart å ha såpass forskjellige attributtnavn i frontend-appen vs. det som APIet forventer. Bedre å sørge for at disse variabelnavnene er like overalt (la APIet bestemme dem) :)